tests/stubs/notifications.ts

```diff
@@ -0,0 +1,9 @@
+import { SinonSandbox } from "sinon";
+import * as notifications from "../../src/notifications";
+
+/**
+ * Stub for the `showErrorNotificationWithButtons` function.
+ */
+export function getShowErrorNotificationWithButtonsStub(sandbox: SinonSandbox) {
```

Comment:
These one-liner stubs haven't caused many problems in the past, so I'm not sure if we really need this one -- I think it's the singleton-pattern stubs and any classes with funky configs that benefit the most from these kinds of convenience functions.